sssd_test_framework/roles/ipa.py

```diff
@@ -641,6 +646,199 @@ def passkey_remove(self, passkey_mapping: str) -> IPAUser:
         self._exec("remove-passkey", [passkey_mapping])
         return self
 
+    '''
+    def idoverrideuser(self, name:str) -> IPAUserIDOverride:
+        user_obj = IPAUser(self.role, name)
+        print("IN the idoverrideuser")
+        return IPAUserIDOverride(self.role, user_obj)
+
+        #return IPAUserIDOverride(self, name)
+    '''
+    def idoverride(self, name: str) -> IDOverride:
+        return IDOverride(self.role, self)
+
+class IDOverride(IPAUser):
+    def __init__(self, role:IPA, user:IPAUser) -> None:
+        super().__init__(role, user.name)
+        #super().__init__(user.role, user.name)
+        self.name = user.name
+
+    def add(self,
+            idview_name:str,
```

Comment:
I'd use just ```name``` instead of ```idview_name```, since it's a different class.

Reply:
Creating new PR for override user subclass, as we have different tickets and different features good to have in seperate PR.